includes/errors/payment-button-error-code.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Razorpay\PaymentButton\Errors;
+
+use Razorpay\Api\Errors as ApiErrors;
+
+class Payment_Button_Error_Code extends ApiErrors\ErrorCode
+{
+    // Razorpay Payment Button
+    const API_PAYMENT_BUTTON_FETCH_FAILED     = 'Razorpay payment button fetch request failed';
```

Comment:
@neera11 is it a payment button or a subscription button? Please change the error message according to that only.

Reply:
changed